OsmAndMapCreator/src/net/osmand/BinaryMerger.java

```diff
@@ -286,6 +302,71 @@ private void combineAddressIndex(String name, BinaryMapIndexWriter writer, Addre
 		writer.endWriteAddressIndex();
 	}
 
+	private void combinePoiIndex(String name, BinaryMapIndexWriter writer, long dateCreated, PoiRegion[] poiRegions, BinaryMapIndexReader[] indexes)
+			throws IOException, SQLException {
+		int readPoiCount = 0;
+		int badNeg = 0;
```

Comment:
unused variables, this trace should be of course in comparator or in test but not in merge.